### ISSUE TITLE
Workaround solution for ‘useWindowAsScrollContainer’ not scrolling in ie/chrome/firefox

### ISSUE THREAD

**Issue by LiuuY** [clauderic/react-sortable-hoc#294]
Come across a problem that using ‘useWindowAsScrollContainer’ property, the window won't scroll when an element dragged to the border of the browser, [#58](https://github.com/clauderic/react-sortable-hoc/issues/58).

It seems that current version v0.6.8 still not fix this problem (maybe it is a feature?).

So, I make a workaround solution:

```javascript
handleMove(e) {
    if (window.innerHeight - event.clientY < 50) {
       window.scrollBy(0, 10)} // you can use throttle here
    } else if (event.clientY < 50) {
       window.scrollBy(0, -10)
    }
 }
```

and bind `handleMove` to `onSortMove` property.

**Comment by ThanosDi** [comment]
While this indeed scroll the window it has the same effect as moving the wheel mouse while dragging an element. The element "placeholders" are relative to the height while starting dragging and not to the current height of the window.
I attached a gif for better demonstration. 
![scroll window](https://user-images.githubusercontent.com/6170355/31778215-c44edcec-b4f8-11e7-8626-d1507ed5ebae.gif)

**Comment by jimyaghi** [comment]
https://github.com/clauderic/react-sortable-hoc/pull/287

**Comment by ambewas** [comment]
@jimyaghi  would you have any idea of when this PR could make it to a release? Scrolling does not work at all at the moment in chrome. 

If there's no foreseeable timeline, the project I'm working on right now is asking me to fork this component. I'd rather not introduce a lingering fork, so that's why I'm asking :-)

**Comment by jimyaghi** [comment]
@ambewas If you use Yarn, this command works for me in Yarn 1.0.1: 

yarn add clauderic/react-sortable-hoc#0299bf685b241ed87b9e3aee45d0d9db01cd1cd3

Sorry i dont know the equivalent in NPM but quite possibly it looks like this:

npm i -S clauderic/react-sortable-hoc#0299bf685b241ed87b9e3aee45d0d9db01cd1cd3

Let me know if that works out.

~jim

**Comment by ThanosDi** [comment]
No I tried that during my tests.
```
$ npm i -S clauderic/react-sortable-hoc#0299bf685b241ed87b9e3aee45d0d9db01cd1cd3
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git checkout 0299bf685b241ed87b9e3aee45d0d9db01cd1cd3
npm ERR! fatal: reference is not a tree: 0299bf685b241ed87b9e3aee45d0d9db01cd1cd3
npm ERR! 

```

**Comment by NSLS** [comment]
@jimyaghi Just tested this on my project. Autoscroll works great when scrolling down, but doesn't work at all when I try to scroll back up

**Comment by clauderic** [comment]
`useWindowAsScrollContainer ` was fixed in `0.7.0` thanks to #306, if you still experience issues with this, feel free to reopen
